biz.aQute.launchpad.tests/bnd.bnd

```diff
@@ -8,4 +8,5 @@
     slf4j.api, \
     slf4j.simple, \
 	${junit},\
-	biz.aQute.launchpad;version=latest
+	biz.aQute.launchpad;version=latest, \
+	aQute.libg
```

Comment:
This should be `aQute.libg;version=project` to be consistent with other projects.